src/main/java/core/mvc/JspViewResolver.java

```diff
@@ -0,0 +1,16 @@
+package core.mvc;
+
+import core.mvc.view.JspView;
+
+public class JspViewResolver implements ViewResolver {
+
+    public static final String JSP_SUFFIX = ".jsp";
+
+    @Override
+    public View resolveViewName(String viewName) {
+        if(viewName.endsWith(JSP_SUFFIX)) {
```

Comment:
공백, 공백라인 컨벤션 등의 부분들은 습관적으로 반영하시는 것이 좋다고 생각해요 😊
```suggestion
        if (viewName.endsWith(JSP_SUFFIX)) {
```